### ISSUE TITLE
extend readme with important information

### ISSUE THREAD

**Issue by jose1711** [mbrt/ruplicity-fuse#6]
hello,

i think it would be useful to make the following notes in the readme:

- directory structure can be browsed, but reading of file contents is not implemented
- gpg-encrypted support (which is often seen with duplicity backups) is missing - i wonder how difficult would be to add this one

thank you,

jose

**Comment by mbrt** [comment]
I agree with you. The limitations come from the underlying ruplicity crate. In particular mbrt/ruplicity#30 and mbrt/ruplicity#16. I was going to merge these two repositories because there is a lot of duplications, especially in the documentation.
